src/descriptor_runner/webdnn.ts

```diff
@@ -195,7 +200,7 @@ export async function load(directory: string, initOption: InitOption = {}): Prom
         runner.ignoreCache = Boolean(initOption.ignoreCache);
 
         try {
-            await runner.load(directory, initOption.progressCallback);
+            await runner.load(directory, initOption.progressCallback, initOption.weightDirectory);
```

Comment:
Please use `registerTransformUrlDelegate` to load binary file from `initOption.weightDirectory` .